pycsob/client.py

```diff
@@ -17,6 +21,213 @@ def send(self, request, **kwargs):
+            value = value[:self._max_length[name]].rstrip()
+        return value
+
+    def _to_camel_case(self, name: str) -> str:
+        """Convert name to camel case form."""
+        parts = name.split("_")
+        return parts[0] + "".join(i.title() for i in parts[1:])
+
+    def to_dict(self) -> Dict[str, Union[str, Dict]]:
+        """Carry out conversion."""
+        data = []
+        get_fields = reorder_fields if hasattr(self, '_ordered_fields') else fields
+        for field in get_fields(self):
+            value = getattr(self, field.name)
+            if value in conf.EMPTY_VALUES:
+                continue
+            if hasattr(value, 'to_dict'):
+                value = value.to_dict()
+            data.append((self._to_camel_case(field.name), self._format_field(field.name, value)))
+        return OrderedDict(data)
+
+
+@dataclass
+class CartItem(ConvertMixin):
+    """Cart item for creating card payment."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Basic-methods#item---cart-item-object-cart-
+
+    name: str
+    quantity: int
+    amount: int
+    description: Optional[str] = None
+
+    _max_length = {"name": 20, "description": 40}
+
+
+@dataclass
+class CustomerAccount(ConvertMixin):
+    """Customer account data."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Purchase-metadata#customeraccount-data-
+
+    created_at: Optional[datetime] = None
+    changed_at: Optional[datetime] = None
+    changed_pwd_at: Optional[datetime] = None
+    order_history: Optional[int] = None
+    payments_day: Optional[int] = None
+    payments_year: Optional[int] = None
+    oneclick_adds: Optional[int] = None
+    suspicious: Optional[bool] = None
+
+
+@unique
+class CustomerLoginType(Enum):
+    """Type of customer login."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Purchase-metadata#customerlogin-data-
+
+    GUEST = "guest"
+    ACCOUNT = "account"
+    FEDERATED = "federated"
+    ISSUER = "issuer"
+    THIRDPARTY = "thirdparty"
+    FIDO = "fido"
+    FIDO_SIGNED = "fido_signed"
+    API = "api"
+
+
+@dataclass
+class CustomerLogin(ConvertMixin):
+    """Customer login data."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Purchase-metadata#customerlogin-data-
+
+    auth: Optional[CustomerLoginType] = None
+    auth_at: Optional[datetime] = None
+    auth_data: Optional[str] = None
+
+
+@dataclass
+class Customer(ConvertMixin):
+    """Customer data for creating card payment."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Purchase-metadata#customer-data-
+
+    name: str
+    email: Optional[str] = None
+    home_phone: Optional[str] = None
+    work_phone: Optional[str] = None
+    mobile_phone: Optional[str] = None
+    account: Optional[CustomerAccount] = None
+    login: Optional[CustomerLogin] = None
+
+    _max_length = {"name": 45, "description": 100}
```

Comment:
Field `description` is not in the class.

Reply:
Fixed as suggested.